app/mailers/user_mentee_application_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class UserMenteeApplicationMailer < ApplicationMailer
+  def notify_for_application_submission
+    @user_mentee_application = params[:user_mentee_application]
```

Comment:
Do you even need the `@user_mentee_application` in this mailer? Doesn't seem the template is using it.

Reply:
actually, I wanted to add a link to the application in the mailer template like this ```  
<p>While you wait, feel free to <%= link_to 'explore your application', user_mentee_application_url(@user_mentee_application) %> for more educational opportunities.</p> ``` 
but for whatever reason rails is not recognizing the route 🤷‍♂️ ...